### DISCUSSION TITLE
Split testing

### DISCUSSION THREAD

**Discussion by rad182** [vitest-dev/vitest#1339] [Q&A]
Hello, I just migrated my project test runner from jest to vitest. But I couldn't see any how to enable split testing similar to jest?
reference: https://circleci.com/blog/a-guide-to-test-splitting/

**Comment by AlexYuleUSDS** [comment]
Did you ever get this sorted? Also wondering

**Reply by hi-ogawa** [reply]
Can you provide more detail on what you're trying to do? A few questions I'd like to check:
- which ci platform? (github action, circle ci)
- are you trying to do time-based splitting as explained in https://circleci.com/blog/a-guide-to-test-splitting/ ?
- would simple random shuffling/splitting via `--shard` option suffice for your use case? https://vitest.dev/guide/cli.html#shard
